src/main/java/Juke.java

```diff
@@ -0,0 +1,81 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+//import java.util.regex.*;
+
+public class Juke {
+
+    // HELPER FUNCTIONS
+    public static void listTasks(ArrayList<Task> tasks) {
+        String line = "------------------------\n";
+        System.out.println(line);
+        for(int i = 0; i < tasks.size(); i++){
+            int j = i + 1;
+            System.out.println(j + ". " + tasks.get(i).getDescription() + tasks.get(i).getStatusIcon() + "\n");
+        }
+        System.out.println(line);
+    }
+
+    // MAIN FUNCTION
+    public static void main(String[] args) {
+        
+        // INITIAL PRINTING
+        String line = "------------------------\n";
+        System.out.println(line + "Hello! I'm Juke!\nWhat can I do for you?\n" + line);
+        
+        // VARS
+        String userInput = "";
+        Scanner scanObj = new Scanner(System.in);
+        ArrayList<Task> tasks = new ArrayList<Task>();
+        
+        // JUKE LOGIC
+        while(!userInput.equals("bye")){
```

Comment:
I think it would be better to classify the `userInput.equals("bye")` as a boolean variable. For example `isBye = userInput.equals("bye");`  Would be easier for the reader to understand.